src/main/scala/org/splink/pagelets/ResourceActions.scala

```diff
@@ -29,21 +30,20 @@ trait ResourceActionsImpl extends ResourceActions { self: Resources with BaseCon
   }
 
   def CacheHeaders(fingerprint: String, validFor: Duration = 365.days) = {
-    val format = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz")
-    format.setTimeZone(TimeZone.getTimeZone("GMT"))
+    def format(zdt: ZonedDateTime) =
+      DateTimeFormatter.RFC_1123_DATE_TIME.format(zdt)
 
-    val now = new Date()
-    val futureDate = Calendar.getInstance()
-    futureDate.add(Calendar.DATE, validFor.toDays.toInt)
+    val now = ZonedDateTime.now(ZoneId.of("GMT"))
+    val future = now.plusDays(365)
```

Comment:
Shouldn't this use `validFor.days` here ?

Reply:
Oh, yes, right. Thanks Martin :)